CHANGELOG.md

```diff
@@ -19,6 +19,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Spying events interface is updated to enable the use of events defined inside contracts in assertions
 
+### Cast
+
+#### Changed
+
+- dropped official support for cairo 1 contracts. while they still should be working without any problems, from now on 
+the only officially supported cairo version is 2
```

Comment:
I would rephrase it because I'm not sure that we are now on Cairo 2 😅 It would be better to call it `cairo compiler version 2` or something like that